exercises/ocr-numbers/ocr_numbers_test.rb

```diff
@@ -3,187 +3,112 @@
 require 'minitest/autorun'
 require_relative 'ocr_numbers'
 
-class OCRTest < Minitest::Test
-  def test_recognize_zero
-    text = <<-NUMBER.chomp
- _ 
-| |
-|_|
-   
-    NUMBER
-    assert_equal '0', OCR.new(text).convert
+# Common test data version: 80782b6
+class OcrNumbersTest < Minitest::Test
+  def test_recognizes_0
+    # skip
+    assert_equal "0", OcrNumbers.convert(" _ \n| |\n|_|\n   ")
```

Comment:
It would be nice to do something like in the previous version that human readably formatted the input data, but that is not a blocker for this PR.